packages/profiling-node/bindings/cpu_profiler.cc

```diff
@@ -538,11 +557,15 @@ napi_value CreateSample(const napi_env &env, const uint32_t stack_id,
                           NAPI_AUTO_LENGTH, &thread_id_prop);
   napi_set_named_property(env, js_node, "thread_id", thread_id_prop);
 
-  napi_value elapsed_since_start_ns_prop;
-  napi_create_int64(env, sample_timestamp_us * 1000,
-                    &elapsed_since_start_ns_prop);
-  napi_set_named_property(env, js_node, "elapsed_since_start_ns",
-                          elapsed_since_start_ns_prop);
+  if (format == ProfileFormat::kFormatThread) {
```

Comment:
m: given this is an enum, I think we should prefer `switch` statement for this. We can also handle non-default format as well then.